prime-dotnet-webapi/Services/BusinessEventService.cs

```diff
@@ -225,6 +225,15 @@ private async Task<BusinessEvent> CreateSiteBusinessEvent(int BusinessEventTypeC
             var userId = _httpContext.HttpContext.User.GetPrimeUserId();
             Admin admin = await _adminService.GetAdminAsync(userId);
             int? adminId = admin?.Id;
+            var firstName = await _context.Parties
+                .Where(p => p.Id == partyId)
+                .Select(p => p.FirstName)
+                .SingleOrDefaultAsync();
+            var lastName = await _context.Parties
+                .Where(p => p.Id == partyId)
+                .Select(p => p.LastName)
+                .SingleOrDefaultAsync();
+            var partyName = String.Format("{0} {1}", firstName, lastName);
```

Comment:
You don't need to go to the DB twice here, you could just do 
```C#
var partyName = await _context.Parties
    .Where(p => p.Id == partyId)
    .Select(p => $"p.FirstName p.LastName")
    .SingleOrDefaultAsync();
```

Reply:
moved the PartyName building logic to the new ViewModel mapping